modules/upgrade-49-acknowledgement.adoc

```diff
@@ -8,7 +8,7 @@
 [id="upgrade-49-acknowledgement_{context}"]
 = Administrator acknowledgment when upgrading to OpenShift 4.9
 
-{product-title} 4.9 uses Kubernetes 1.22, which removed a xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[significant number of deprecated `v1beta1` APIs].
+{product-title} 4.9 uses Kubernetes 1.22, which removed a link:https://docs.openshift.com/container-platform/4.9/release_notes/ocp-4-9-release-notes.html#ocp-4-9-removed-kube-1-22-apis[significant number of deprecated `v1beta1` APIs].
```

Comment:
I am changing this to a link because otherwise the following build error is seen for OSD:

`ERROR (modules/upgrade-49-acknowledgement.adoc): "xref:../release_notes/ocp-4-9-release-notes.adoc#ocp-4-9-removed-kube-1-22-apis[significant number of deprecated `v1beta1` APIs]" appears to try to reference a file not included in the "openshift-dedicated" distro`